tests/test_dataset.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
```

Comment:
move file to `tests/sparsezoo/utils/test_data.py` to match file naming conventions